src/App.jsx

```diff
@@ -1,43 +1,26 @@
-import React, { Component } from "react";
+import React, { useState } from "react";
```

Comment:
Can delete "React" as it's not used. Should be

import { useState } from "react";

Always remove unused imports